content/docs/help/contact_us.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Contact Us"
+description = "Contact mSupply Foundation"
+date = 2025-01-06T11:45:00+00:00
+updated = 2025-01-06T11:45:00+00:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 1
+draft = false
+
+[extra]
+toc = true
+top = false
++++
+
+The contact form allows you to reach out to us with any feedback or support requests.
+
+## Viewing the contact form
+
+To view the contact form, go to `Help` in the lower section of the navigation panel:
+
+![Help: nav](/docs/help/images/help_nav.png)
```

Comment:
Would put a `This will take you to the Help page` with a full page screenshot after this